### ISSUE TITLE
GvrTrackedController - GVR 1.100

### ISSUE THREAD

**Issue by andyandrewdrew** [googlevr/gvr-unity-sdk#746]
In the OnDisable() function, add this line:
`GvrControllerInput.OnStateChanged -= OnControllerStateChanged;`
 
Or, in the OnEnable()( function, remove the existing line: 
`GvrControllerInput.OnStateChanged -= OnControllerStateChanged;`

We subscribe to OnControllerStateChanged in the Awake and OnEnable functions and unsubscribe only in the OnDestroy function. I found this bug when I change scenes. In the console, I get the following error line: "MissingReferenceException: The object of type 'GvrTrackedController' has been destroyed but you are still trying to access it."

**Comment by fredsa** [comment]
Found in Google VR SDK for Unity 1.100.0.

**Comment by fredsa** [comment]
For search-ability, including the error message seen in the Android device logs (`adb logcat -s Unity`) after switching scenes using Google VR SDK for Unity 1.100.0:

```
NullReferenceException
  at GvrTrackedController.OnControllerStateChanged (GvrConnectionState state, GvrConnectionState oldState) in Assets/GoogleVR/Scripts/Controller/GvrTrackedController.cs:118
  at GvrControllerInput.Update () in Assets/GoogleVR/Scripts/Controller/GvrControllerInput.cs:491
  at GvrControllerInput.get_Gyro () in Assets/GoogleVR/Scripts/Controller/GvrControllerInput.cs:204
  at GvrArmModel.UpdateTorsoDirection () in Assets/GoogleVR/Scripts/Controller/ArmModel/GvrArmModel.cs:236
  at GvrArmModel.OnControllerInputUpdated () in Assets/GoogleVR/Scripts/Controller/ArmModel/GvrArmModel.cs:210
  at GvrArmModel.OnEnable () in Assets/GoogleVR/Scripts/Controller/ArmModel/GvrArmModel.cs:201
```

**Comment by rusmaxham** [comment]
We're working on a fix now.

**Comment by rusmaxham** [comment]
Fixed in 1.100.1.
